### ISSUE TITLE
inference is not allowed

### ISSUE THREAD

**Issue by brucehoff** [Sage-Bionetworks/DigitalMammographyChallenge#245]
https://www.synapse.org/#!Synapse:syn4224222/discussion/threadId=1247

**Comment by tschaffter** [comment]
Hypothesis: initially it was "scoring=" but we decided to change that for "inference=", which is used in the documentation. Is "inference=" used in the code?

**Comment by brucehoff** [comment]
> Is "inference=" used in the code?
No.  You can see the property file 'key' definitions at ll. 14-15 here:

https://github.com/Sage-Bionetworks/ChallengeDockerAgent/blob/master/src/main/java/org/sagebionetworks/ScoringSubmissionParameters.java

**Comment by brucehoff** [comment]
To avoid confusion let's not change the published instructions.  I will update the code to allow either 'scoring=' or 'inference='.

**Comment by brucehoff** [comment]
We have to restart the following submissions, erroneously labeled INVALID:
7684960 mobileroaming
7740820 Dezso Ribli (riblidezso)
7741066 Dezso Ribli (riblidezso)
7747683 Aashish Dattani (zettata)

**Comment by brucehoff** [comment]
I examined all the submissions marked invalid, including the ones above.  They were all either truly invalid or deleted by the participant. (detailed below)  I notified the participants that we have corrected the problem with our pipeline and they can resubmit.

prod-SC1
7674528 ACTUALLY INVALID
7684982 ACTUALLY INVALID
7740832 **deleted**

prod-SC2
7687812 ACTUALLY INVALID

EL-SC1
7684960 ACTUALLY INVALID
7740820 **deleted**
7741066 <<< This is a Docker container! / later deleted
7747683 ACTUALLY INVALID

EL-SC2
7675285 <<< This is a Docker container!  I verified that containers can be submitted for scoring.
7687792 ACTUALLY INVALID
